Singer.API/Services/CareUserService.cs

```diff
@@ -19,12 +19,7 @@ public class CareUserService : UserService<CareUser, CareUserDTO, CreateCareUser
    {
       protected override DbSet<CareUser> DbSet => Context.CareUsers;
 
-      protected override IQueryable<CareUser> Queryable => Context.CareUsers
```

Comment:
Removing this broke it.
By not using the `.Include`, values can become null when using `.Single` or alike when they're not explicitly loaded. When using LINQ-queries they can be ommitted though.